src/components/navigation/Navbar.tsx

```diff
@@ -1,10 +1,69 @@
 import React from "react";
 import { Link } from "react-router-dom";
+import { GiSpaceShuttle } from "react-icons/gi";
+import {
+  IoMdPlanet,
+  IoIosCamera,
+  IoMdCalendar,
+  IoIosInformationCircleOutline,
+  IoMdMenu,
+} from "react-icons/io";
+import "./NavBar.scss";
+import Hamburger from "./Hamburger";
+import react, { useState, useEffect, useRef } from "react";
```

Comment:
Can you add these import statements to line 1 and remove any unused imports?

Reply:
Removed duplicates & tidied